pkg/docker/base_image_test.go

```diff
@@ -40,12 +49,21 @@ func TestValidImage(t *testing.T) {
 		Paths: []string{"./fixtures/compose-image"},
 	}
 	c.RunCheck()
-	if msg, ok := internal.EnsurePasses(t, &c.CheckBase, []string{"Dockerfiles adhere to the policy"}); !ok {
-		t.Error(msg)
-	}
+	assert.Equal(shipshape.Pass, c.Result.Status)
+	sort.Strings(c.Result.Passes)
+	assert.EqualValues(
```

Comment:
you can use `assert.ElementsMatch` instead of the sort followed by EqualValues:

```suggestion
	assert.ElementsMatch(
```